content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md

```diff
@@ -10,4 +10,59 @@ compatibility.  We also mention cases where backwards compatibility was
+
+## Citadel
+
+Previously, Citadel served two functionalities: writing certificates to secrets in each namespace, and serving secrets to the `nodeagent` over `gRPC` when SDS is used. In Istio 1.5, secrets are no longer written to each namespace. Instead, they are only served over gRPC. This functionality has been moved to the `istiod` deployment.
+
+## SDS Node Agent
+
+The `nodeagent` deployment has been removed. This functionality now exists in the Sidecar
+
+## Sidecar
+
+Previously, the sidecar was able to access certificates in two ways: through secrets mounted as files, or over SDS (through the `nodeagent` deployment). In Istio 1.5, this has been simplified. All secrets will be served over a locally run SDS server. For most users, these secrets will be fetched from the `istiod` deployment. For users with a custom CA, file mounted secrets can still be used, however, these will still be served by the local SDS server. This means that certificate rotations will no longer require Envoy to restart.
+
+## CNI
+
+There have been no changes to the deployment of `istio-cni`.
+
+## Pilot
+
+The `istio-pilot` deployment has been removed in favor of the `istiod` deployment, which contains all functionality that Pilot once had. For backwards compatibility, there are still some references to Pilot.
+
+# Mixer Deprecation
+
+Mixer is deprecated in Istio 1.5. The default profile in Istio 1.5 switches off `mixer` completely. Telemetry V2 is the default in Istio 1.5 and it does not require `mixer`.
+
+If you depend on specific Mixer features like out of process adapters, you may re-enable Mixer. Mixer will continue receiving bug fixes and security fixes until Istio 1.7.
+Many features supported by `mixer` have alternatives as specified in the [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) document including the [in-proxy extensions](https://github.com/istio/proxy/tree/master/extensions) based on the WebAssembly sandbox API.
+
+If you rely on a Mixer feature that does not have an equivalent, we encourage you to open issues and discuss in the community.
+
+Please check [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) notice for details.
+
+# Control Plane Security
+
+As part of the Istiod effort, we have changed how proxies securely communicate with the control plane. In previous versions, proxies would connect to the control plane securely when the setting `values.global.controlPlaneSecurityEnabled=true` was configured, which was the default for Istio 1.4. Each control plane component ran a sidecar with Citadel certificates, and proxies connected to Pilot over port 15011.
+
+In Istio 1.5, this is no longer the recommended or default way to connect the proxies with the control plane; instead, DNS certificates, which can be signed by Kubernetes or Istiod, will be used without a sidecar. Proxies connect to pilot over port 15012.
```

Comment:
It might help make this clearer to non-security savvy folk if we added some more detail. *Who/What*  will use the DNS certs without a sidecar -- and how?